python_functions_more_fun_with_functions.py

```diff
@@ -0,0 +1,14 @@
+my_lipsum_str = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum consectetur nibh ex, quis pretium risus faucibus eu. Ut placerat lorem vel sapien vestibulum consequat. Aliquam sit amet lacinia turpis, eget congue orci. Nulla faucibus, massa sed ultricies consectetur, arcu massa aliquam ante, id egestas ligula ligula ornare nunc. Mauris euismod, lorem non maximus sagittis, sapien justo. "
+
+def num_of_letters(a):
+    num_upper = sum(1 for letter in a if letter.isupper()) #A thread that helped me: https://stackoverflow.com/questions/18129830/count-the-uppercase-letters-in-a-string-with-python 
```

Comment:
To je pekny napad. I spolu s tim doplnkem. Avsak pozor!

len("my string") = 9, num_upper = 0, ale kolik je tam malych pismen? Neni jich devet, ale 8 a mezera.

Proto je lepsi testovat kod na mensim vstupu nez hned pouzit dlouhy Lorem.